pom.xml

```diff
@@ -469,6 +480,31 @@
           <useSystemClassLoader>false</useSystemClassLoader>
         </configuration>
       </plugin>
+
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>license-maven-plugin</artifactId>
+        <version>2.0.0</version>
+        <executions>
+          <execution>
+            <configuration>
+              <verbose>false</verbose>
+              <thirdPartyFilename>THIRD-PARTY-Licenses.txt</thirdPartyFilename>
+              <useMissingFile>true</useMissingFile>
+              <missingFile>THIRD-PARTY-Missed-Licenses.txt</missingFile>
+              <deployMissingFile>true</deployMissingFile>
+            </configuration>
```

Comment:
[To the best of my knowledge](https://github.com/search?q=org%3Afolio-org+filename%3ATHIRD-PARTY.txt&type=code), this is the first example of adding a THIRD-PARTY.txt file to a `folio-org` repository, which was to fulfill the requirements in the comments of [MODINVSTOR-381](https://issues.folio.org/browse/MODINVSTOR-381).  In principle, I don't have a problem with using [License Maven Plugin – license:add-third-party](https://www.mojohaus.org/license-maven-plugin/add-third-party-mojo.html) to automate this task.  It does make me wonder if this is something that is needed for all the back-end modules.  [I don't see an issue tracking this need](https://issues.folio.org/issues/?jql=text%20~%20%22THIRD-PARTY%20license%22).  Should there be one?

That said, I endorse @julianladisch's request above to use the common file name conventions unless there is a reason to do otherwise.

Reply:
https://issues.folio.org/browse/FOLIO-3046 "Add THIRD-PARTY.txt to generated jars"